Content.Server/Language/DialectPrototype.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Language
+{
+    [Prototype("dialect")]
```

Comment:
Dialect is not an easily understood name for what this is used for (replacing the old functionality of replacement accents)

Reply:
I tried for a while to think of a better name for "something that makes you sound different to people who do not know what language you're speaking without invoking the default distorter."

Its only use case is to allow animals to sound different while speaking the same language. I thought that I could just do away with Dialects entirely and make all animals understand each others' animal language while only being able to speak their respective animal language. That's a possible solution which would simplify a lot of this.

Dialects are not meant to replace accents, per se. Only to those who do not understand the language. Accents as they are should still work to alter the speaker's text, like the rat king hat, without completely changing the language.

Reply:
Use "unintelligible sounds" or something

Reply:
That works.